src/server.h

```diff
@@ -850,6 +850,7 @@ typedef struct client {
     dict *pubsub_channels;  /* channels a client is interested in (SUBSCRIBE) */
     list *pubsub_patterns;  /* patterns a client is interested in (SUBSCRIBE) */
     sds peerid;             /* Cached peer ID. */
+    sds sockname;
```

Comment:
Redis generally documents the datastructure, so you can just look at it to understand the implementation.

Reply:
:+1: